language/de/install.php

```diff
@@ -570,6 +572,10 @@
 	'CONVERT_COMPLETE'			=> 'Konvertierung abgeschlossen',
 	'CONVERT_COMPLETE_EXPLAIN'	=> 'Dein Board wurde erfolgreich auf phpBB 3.3 konvertiert. Du kannst dich jetzt anmelden und <a href="../">dein Board betreten</a>. Bitte prüfe, ob alle Einstellungen richtig übernommen wurden, bevor du dein Board durch Löschen des „install“-Verzeichnisses freigibst. Hilfe zum Gebrauch von phpBB erhältst du online über die <a href="https://www.phpbb.com/support/docs/en/3.3/ug/">Dokumentation (englisch)</a> (<a href="https://www.phpbb.de/go/3.3/dokumentation">deutsche Übersetzung</a>) und das <a href="https://www.phpbb.com/community/viewforum.php?f=661">Support-Forum (englisch)</a> (<a href="https://www.phpbb.de/go/3.3/supportforum">deutschsprachiges Forum auf phpBB.de</a>).',
 
+	'COLLIDING_CLEAN_USERNAME'			=> '<strong>%s</strong> ist der bereinigte Benutzername für:',
+	'COLLIDING_USER'					=> '» Benutzer-ID: <strong>%d</strong> Benutzername: <strong>%s</strong> (%d Beiträge)',
+	'COLLIDING_USERNAMES_FOUND'			=> 'Es wurden doppelte Benutzernamen in deinem Board gefunden. Um mit der Konvertierung fortzufahren, müssen diese Benutzer gelöscht oder umbenannt werden, so dass in deinem alten Board nur ein Benutzer mit jedem bereinigtem Benutzernamen existiert.',
```

Comment:
Etwas freier: so dass in deinem alten Board jeder bereinigte Benutzername nur für einen Benutzer (alternativ: einmal) verwendet wird.